index.md

```diff
@@ -613,51 +613,66 @@ so it is worth checking with your IT department.
 
 ## Manuscripts
 
-A common, but unfortunate, practice is that the lead author at different points in the writing process
-sends around versions to coauthors for collecting feedback, and receiving multiple documents back with comments.
-This leads to a lot of files to keep track of and much manual labor
-to merge all the comments into the updated 'master' document.
-Instead, we recommend mirroring the practices we describe for managing software and data
-as they are equally beneficial for the process of manuscript preparation:
-making the writing version controlled, collaborative, and reproducible.
-It is important, however, to have all authors agree on a particular collaborative writing workflow
-*before* starting the actual writing.
-Once you agree, as lead author, don't hesitate to lay out some ground rules
-that you expect the others to adhere to --- even if they are many years your senior.
-
-### Goals
-
-This section describes workflows for manuscript preparation that aim to satisfy the following goals:
-
-1.  Make text accessible to yourself and others now and in the future
-    by using a single point of access for the master document that is accessible to all coauthors at all times.
-2.  Reduce chances of work being lost or people overwriting each other's work.
+Gathering data,
+analyzing it,
+and figuring out what it means is the first 90% of any project;
+writing up is the other 90%.
+While this step is often skipped over,
+computing has changed it just as much as it has changed the research itself.
+
+A common, but unfortunate, practice is for
+the lead author to send successive versions of a manuscript to coauthors to collect feedback,
+which is received as multiple documents with changes,
+as comments on the document,
+in email,
+or a mix of all three.
+The result is a lot of files to keep track of
+and much tedious manual labor to merge all the comments
+to create a new master version,
+at which point the process begins again.
+
+FIXME: following paragraph will need to be changed if version control is rewritten as either/or.
+
+Instead of an email-based workflow,
+we recommend mirroring good practices for managing software and data,
+to make writing scalable, collaborative, and reproducible.
+In contrast to our other recommendations,
+however,
+we suggest that groups choose one of two different approaches for managing manuscripts.
+The most important rule is
+to have all authors agree on one or the other *before* writing starts.
+
+The goals of both of the workflows outlined below are:
+
+1.  Ensure that text accessible to yourself and others
+    now and in the future
+    by making a single master document accessible to all coauthors at all times.
```

Comment:
two times 'accessible' in the same sentence. This one could be change into 'available'?